internal/k8s/names_test.go

```diff
@@ -71,3 +73,50 @@ func TestUniqueResourceNames(t *testing.T) {
 		})
 	}
 }
+
+func TestFragmentsToEntities(t *testing.T) {
+	entities, err := ParseYAMLFromString(testyaml.BlorgBackendYAML)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	actual := FragmentsToEntities(entities)
+
+	expected := map[string][]K8sEntity{
+		"devel-nick-lb-blorg-be":                 {entities[0]},
+		"devel-nick-lb-blorg-be:service":         {entities[0]},
+		"devel-nick-lb-blorg-be:service:default": {entities[0]},
+		// TODO(dmiller): this is wrong
```

Comment:
lol i believe you , but if you merge this w/o fixing, more detail here for those of us who are not dan pls? :) and below as well